deepchem/feat/molecule_featurizers/rdkit_descriptors.py

```diff
@@ -105,8 +105,20 @@ def __init__(self,
                     continue
                 if self.use_bcut2d is False and desc_name.startswith('BCUT2D_'):
                     continue
+                if desc_name == 'AvgIpc':  # already handled by 'Ipc' during featurization
+                    continue
                 self.reqd_properties[desc_name] = function
         else:
+            if self.ipc_avg is True:
```

Comment:
Can you add a unit test for these new edge cases?

Reply:
sir I have added the test